src/Orleans/Serialization/SerializationManager.cs

```diff
@@ -2210,40 +2186,5 @@ public SerializerMethods(DeepCopier deepCopy, Serializer serialize, Deserializer
             public Serializer Serialize { get; private set; }
             public Deserializer Deserialize { get; private set; }
         }
-
-        public static JsonSerializerSettings GetDefaultJsonSerializerSettings()
```

Comment:
See the other comment about moving the method. If you feel it's better to keep the methods on SerializationManager, then I can move them back. It seems messy, though.

A cleaner approach would be for those storage providers to consume an `IExternalSerializer` for all of their serialization, rather than each implementing support for JSON + Orleans serialization.